src/shipit_pulse_listener/tests/test_code_coverage_listener.py

```diff
@@ -83,6 +83,15 @@ def test_is_old_task():
             }]
         }
     }
+    assert not hook.is_old_task(old_task)
```

Comment:
Nit: Isn't it weird to assert that something called `old_task` is *not* an old task?